web/src/settings_config.ts

```diff
@@ -77,11 +77,10 @@ export const user_list_style_values = {
         code: 2,
         description: $t({defaultMessage: "Show status text"}),
     },
-    // The `with_avatar` design in still in discussion.
-    // with_avatar: {
-    //     code: 3,
-    //     description: $t({defaultMessage: "Show status text and avatar"}),
-    // },
+    with_avatar: {
+        code: 3,
+        description: $t({defaultMessage: "Show avatar"}),
+    },
```

Comment:
A strategy we can take for this PR will be to make this logic only add the last option in a development environment; then we can merge the main PR and iterate on smaller changes.

Reply:
I think it could be helpful to test it on CZO sooner than later, which leads me to not want to restrict it to a development environment. I think right now the only remaining things are (1) slow loading (2) tweaks to the avatar size and text line-height.

Reply:
It's trivial to remove the development-only hack for a CZO deployment; the point is to be able to merge to `main` without potentially breaking production.

Reply:
Ah okay, cool